plugins/collections/create.js

```diff
@@ -21,6 +21,19 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
+            // Check if the collection to be created has a type 'default'
+            if (request.payload.type === 'default') {
+                return reply(boom.forbidden(
+                    'Collection with type "default" cannot be created by user'
+                ));
+            }
+
+            // If the request doesn't specify the collection type
+            // a 'normal' type is added to the payload
+            if (!request.payload.type) {
```

Comment:
if request.payload.type is either undefined or not part of allowed types, then default it to `normal`